include/export.bash

```diff
@@ -0,0 +1,122 @@
+export-logs() {
+    declare desc="Exports and anonymizes logs for fault analysis. Usage: cbd export-logs [archive_name]"
+    declare archivename=$1
+    if [[ -z $archivename ]] || [[ "/" == $archivename ]]; then
+        archivename="cbd_export_$(date +%s)"
+        warn "Invalid archive filename. Generated filename: $archivename"
+    fi
+
+    local collectdir="collected_tmp"
+    mkdir -p $collectdir 
+
+    export-psql $collectdir
+    export-prerequisites $collectdir
+
+    export-cbd-env-show $collectdir
+    export-cbd-doctor $collectdir
+    export-cbd-version $collectdir
+    export-cbd-ps $collectdir
+
+    anonymize-exported-files $archivename
+
+    archive-directory $archivename $collectdir
+}
+
+export-psql() {
+    local psqlcmd="docker exec -t cbreak_commondb_1 cat /var/lib/postgresql/.psql_history"
+
+    ! $($psqlcmd) > "$1/.psql_history"
+    check-export-success "$1/.psql_history"
+}
+
+export-prerequisites() {
+    local out="$1/prerequisites.out"
+
+    if command_exists getenforce; then
+        printf "\n### SELinux\n" >> $out
+        ! echo SELinux status: $(getenforce) >> $out
+    else
+        warn "Command not supported: getenforce"
+    fi
+
+    if command_exists ifconfig; then
+        printf "\n### MTU Sizes\n" >> $out
+        ! ifconfig | grep mtu | awk {'print $1 " " $NF'} >> $out
+    else
+        warn "Command not supported: ifconfig"
+    fi
+
+    if command_exists netstat && [[ $(uname) != "Darwin" ]]; then
```

Comment:
We have ismacos function and islinux too

Reply:
I suggest to use container with for example lsof and we don't need to use multiple commands

Reply:
To your second comment: how would an `lsof` executing inside a container could give information about the host connections?